types/react-jsonschema-form/index.d.ts

```diff
@@ -23,6 +25,7 @@ declare module "react-jsonschema-form" {
         liveValidate?: boolean;
         safeRenderCompletion?: boolean;
         FieldTemplate?: any;
+        transformErrors?: (errors: any) => any;
```

Comment:
`errors` will always be an array of `{ name, message }`, right? It would be a nice enhancement to capture that, but not a blocker.

Reply:
Thanks for approval 
I'll add that enhancement in the coming pr